old-ui/app/app.js

```diff
@@ -48,7 +48,7 @@ const ConfirmChangePassword = require('./components/confirm-change-password')
 const ethNetProps = require('eth-net-props')
 const { getMetaMaskAccounts } = require('../../ui/app/selectors')
 const { getNetworkID } = require('./util')
-const RifModal = require('../../ui/app/rif/components/modal/rifModal')
+const modal = require('../../ui/app/rif/components/modal/modal')
```

Comment:
In general in js, if you have this, like modal/modal where the first one is a folder and the last one is a js file, you can use index.js instead of modal.js, then the require should be require ('../../ui/app/rif/components/modal') instead and you don't have repeated names.